vmware/vsphere_provider.yml

```diff
@@ -24,6 +24,6 @@ spec:
     name: vsphere-provider-secret
     namespace: konveyor-forklift
   settings:
-    vddkInitImage: 'quay.io/kubevirt/vddk-test'
+    vddkInitImage: 'localhost:5001/vddk-test-vmdk'
```

Comment:
maybe it's the right time to make vddk optional? we thought about making it optional so users won't *have to* create the vddk image but if there's no value from using vddk in this test, we could also apply it here..

Reply:
if we dont provide any vddk at all ,then  the cdi importer will fail  on the ndbkit part.
 https://github.com/kubevirt/containerized-data-importer/blob/46c6aa994aec9267532e56d84dfa4deca5d0aefe/pkg/importer/vddk-datasource_amd64.go#L89

Reply:
ack, I think you told me that before but now I see what you meant - ok, so what I suggested above can only be done when delegating the disks retrieval to virt-v2v (rather than CDI) then..